lib/streamlit/elements/arrow_altair.py

```diff
@@ -978,8 +987,8 @@ def _generate_chart(
     chart = alt.Chart(
         data=df,
         mark=chart_type.value["mark_type"],
-        width=width,
-        height=height,
+        width=width or 0,
+        height=height or 0,
```

Comment:
@LukasMasuch I'm copying these edits into my PR, but I'm not sure what this line is and I'm a little less confident copying into where I think it goes. I found the built_in_chart_utils.py file and I updated the par in `def _generate_chart` to have `int | None = None`. What is this part exactly that previously assigned an int and is not assigning a bool?

Reply:
You could either use the `width or 0` whenever `generate_chart` is called in `vega_charts.py`, or allow `None` in `generate_chart` for width and height and add this into the top of `generate_chart` function:

```python
width = width or 0
height = height or 0
```

this basically says that its guaranteed to be a number or 0